features/ouders.feature

```diff
@@ -167,84 +158,86 @@ Rule: de geleverde oudergegevens zijn de gegevens zoals die staan op de persoons
-    | burgerservicenummer | 000007895                                                                               |
+    | burgerservicenummer | 000000218                                                                               |
     | fields              | ouders.ouderAanduiding,ouders.burgerservicenummer,ouders.geslacht,ouders.naam.voornamen |
     Dan heeft de response een persoon met een 'ouder' met de volgende gegevens
     | naam                  | waarde    |
-	| burgerservicenummer   | 000007896 |
+    | burgerservicenummer   | 000007896 |
     | geslacht.code         | V         |
     | geslacht.omschrijving | vrouw     |
-	| naam.voornamen        | Ellen     |
+    | naam.voornamen        | Ellen     |
     | ouderAanduiding       | 1         |
     En heeft de persoon een 'ouder' met de volgende gegevens
     | naam                  | waarde    |
-	| burgerservicenummer   | 000007897 |
+    | burgerservicenummer   | 000007897 |
     | geslacht.code         | M         |
     | geslacht.omschrijving | man       |
-	| naam.voornamen        | Karel     |
+    | naam.voornamen        | Karel     |
     | ouderAanduiding       | 2         |
 
 @gba
 Rule: Wanneer alleen gegevens in groep 81, 82, 83, 84, 85 en/of 86 zijn opgenomen en geen gegevens in groep 1, 2, 3, 4 of 62, dan wordt de ouder niet opgenomen
   Gebruik van de fields parameter heeft geen invloed op het leveren van een ouder
 
-  @gba
   Scenario: vondeling (onbekende ouder met ouderaanduiding "1" en geen ouder met ouderaanduiding "2")
-    Gegeven de persoon met burgerservicenummer '000007896' heeft een ouder '1' met de volgende gegevens
+    Gegeven de persoon met burgerservicenummer '000000231' heeft een ouder '1' met de volgende gegevens
     | datum ingang familierechtelijke betrekking (62.10) | geslachtsnaam (02.40) | geslachtsaanduiding (04.10) |
     | 20190614                                           | .                     | V                           |
     En de persoon heeft een ouder '2' met de volgende gegevens
-	| aktenummer (81.20) | ingangsdatum geldigheid (85.10) |
-	| 2•E0001            | 20160518                        |
+    | aktenummer (81.20) | ingangsdatum geldigheid (85.10) |
+    | 2•E0001            | 20160518                        |
     Als personen wordt gezocht met de volgende parameters
     | naam                | waarde                          |
     | type                | RaadpleegMetBurgerservicenummer |
-    | burgerservicenummer | 000007896                       |
+    | burgerservicenummer | 000000231                       |
     | fields              | ouders                          |
     Dan heeft de response een persoon met een 'ouder' met de volgende gegevens
     | naam                                                | waarde       |
     | ouderAanduiding                                     | 1            |
+    | naam.geslachtsnaam                                  | .            |
```

Comment:
de standaardwaarde "." voor geslachtsnaam wordt niet geleverd. Is deze test geslaagd?

Reply:
Nee, want er wordt geen groep 'naam' geleverd met de property 'geslachtsnaam' met de waarde '.'

Ik begrijp uit je reactie dat dat ook niet zou moeten, klopt dat? Moet er een lege 'naam' groep geleverd worden?
Ook in andere scenario's (in andere features) is er volgens mij vanuit gegaan dat een '.' waarde wel wordt uitgeleverd.

Reply:
gba levert de punt, de proxy filter die eruit. zie onbekend_waardes.feature